src/Common/src/Interop/Unix/System.Net.Security.Native/SecuritySafeHandles.cs

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Also the filename should have GSS in it. We will need a similar file for NTLM related SafeHandles